lib/ecb/rates_document.rb

```diff
@@ -0,0 +1,45 @@
+module ECB
```

Comment:
nit: Can you please use the full name here to avoid potential collisions with ECB abbreviation?

Reply:
@antstorm Could not come up with anything better than `EuCentralBank` for the module name so moved `RatesDocument` it to be inside that same class. Didn't want to move the class around as it is already used as is everywhere.